src/clr-angular/data/datagrid/_datagrid.clarity.scss

```diff
@@ -890,12 +825,63 @@
         min-width: 0 !important;
     }
 
-    @include fixForMsEdge {
-        .datagrid .datagrid-column .datagrid-filter-toggle {
-            // MSEdge doesn't round the rem units properly
-            width: 0.59rem;
-            height: 0.59rem;
-            margin-top: -1px;
+    // Used in table-renderer.ts in noLayout with NO_LAYOUT_CLASS.
+    .datagrid-no-layout {
```

Comment:
If you don't mind, can you put the original comment too, to explain this is just for performance optimization when rendering? Here is is:
```typescript
// Class only applied temporarily when preforming internal operations,
// to improve rendering performance.
```

Feel free to edit it, obviously, all that matters is that the comment gives the "why" of these styles.